### ISSUE TITLE
Short audio burst when changing volume while map is loading in

### ISSUE THREAD

**Issue by Morilli** [ppy/osu-stable-issues#124]
Starting a map the first time from song select can result in a short loud audio burst playing when the volume is changed while the map is loading in. This happens to me quite often, as I want to adjust the volume of the map after clicking to play.

Videos showing the issue:
https://streamable.com/x5usq
https://streamable.com/7il1s

**Comment by peppy** [comment]
What is your UO set to?

**Comment by peppy** [comment]
Fixed in next cutting edge

**Comment by Morilli** [comment]
Is this supposed to be fixed in ``Cutting Edge 20191024.1``? I saw this note in the changelog:
![image](https://user-images.githubusercontent.com/35152647/67484285-102e1d00-f668-11e9-9231-818597097063.png)
but I can still reproduce the issue on this cutting edge build.

**Comment by peppy** [comment]
Curious, it fixed it for me. Is it the exact same scenario as depicted in the original videos?

**Comment by Morilli** [comment]
Yes, looks like it. It seems to start the song select preview in that short moment between clicking the map and the map actually loading in. It only works one successive time for each map.

**Comment by Bitslayn** [comment]
Having this issue on b20250815

**Comment by Bitslayn** [comment]
https://github.com/user-attachments/assets/c3cfac66-580d-4d8e-baa1-d356d5d00362
